src/main/java/com/atlassian/bitbucket/jenkins/internal/model/TestResults.java

```diff
@@ -0,0 +1,35 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TestResults {
+
+    private int successful;
+    private int failed;
+    private int ignored;
+
+    public TestResults(int successful, int failed, int ignored) {
+        this.successful = successful;
+        this.failed = failed;
+        this.ignored = ignored;
+    }
+
+    @JsonProperty(value = "successful")
+    public int getSuccessful() {
```

Comment:
Nullable `Integers` instead of `int`s?

Reply:
I don't think so. I have it as ints from Jenkins, and we don't put this structure in if we have no test results. So it is all or nothing